.github/helm/affine-cloud/.gitignore

```diff
@@ -0,0 +1 @@
+charts/
```

Comment:
Redundant changes?

Reply:
charts/ is the dependencies storage path for helm, it can use `helm dependencies build` to restore in ci
similler to `node_modules`